javascript/javascript1/week4/Tic-tac-toe/tictactoe.js

```diff
@@ -0,0 +1,69 @@
+function startGame() {
+
+    for (let i = 1; i <= 9; i++) {
+        clearBox(i);
+    }
+    document.turn = "X";
+    document.winner = null;
+    setMessage(document.turn + " get's to start.")
+}
+
+function setMessage(msg) {
+    document.getElementById("message").innerText = msg;
+}
+
+function nextMove(square) {
+    if (document.winner != null) {
+        setMessage(document.turn + " already won.");
+    } else if (square.innerText == '') {
+        square.innerText = document.turn;
+        switchTurn();
+    } else {
+        setMessage("Pick another square");
+    }
+}
+
+function switchTurn() {
+    if (checkForWinner(document.turn)) {
+        setMessage("congrats " + document.turn + ", you won!!");
+        document.winner = document.turn;
+    } else if (document.turn == "X") {
+        document.turn = "O";
+        setMessage(document.turn + " get's to start.");
+    } else {
+        document.turn = "X"
+        setMessage(document.turn + " get's to start.");
+    }
+}
+
+function checkForWinner(move) {
+    let result = false;
+    if (checkRow(1, 2, 3, move) ||
+        checkRow(4, 5, 6, move) ||
+        checkRow(7, 8, 9, move) ||
+        checkRow(1, 4, 7, move) ||
```

Comment:
Here you are checking a column. and the *checkRow* name is not good. Maybe think of another name for this function. Something like bellow maybe
```suggestion
        checkBoxes(1, 4, 7, move) ||
```

Reply:
Implemented feedback